batch/egpool/egpool.go

```diff
@@ -77,7 +77,7 @@ func (eg *Group) processJobs() {
 	defer func() {
 		if !finished {
 			if p := recover(); p != nil {
-				eg.err(ErrPanic{p})
+				eg.err(PanicError{p})
 			} else {
 				eg.err(ErrGoexit)
```

Comment:
Should this name also be changed? I don't really know what's being changed here. Or is `ErrGoexit` a value, rather than a type?

Reply:
yes, `ErrGoexit` is a variable, which are supposed to start with `Err`